cumulusci/cumulusci.yml

```diff
@@ -564,9 +579,13 @@ orgs:
     scratch:
         dev:
             config_file: orgs/dev.json
+            days: 14
```

Comment:
I think 7 is still an appropriate default, unless we've heard from devs they want longer lasting ones.

Reply:
I changed the default for dev to 7 and the global default from 7 to 1 for all orgs.